lib/src/base-styles/_nx-alert.scss

```diff
@@ -66,11 +66,11 @@
 }
 
 .nx-alert--info {
-  background-color: #f1f7ff;
-  border-color: #045dd1;
+  background-color: $nx-blue-50;
+  border-color: $nx-blue-600;
```

Comment:
I see that these numbers are copied from zeplin, but what do they mean/measure?

Reply:
> I see that these numbers are copied from zeplin, but what do they mean/measure?

I was under the impression that those numbers were essentially arbitrary, but the scale would be consistent between colours so we'd know that towards 1000 is darker and towards 0 is lighter. @dturner32 ?

Reply:
> but the scale would be consistent between colours so we'd know that towards 1000 is darker and towards 0 is lighter.

If that were the case I'd think we'd use adjacent numbers.  Or is the idea to build in gaps in case we want in-between colors in the future?

Reply:
those numbers are a value scale from lightest to darkest, 50 being the lightest and 1000 being the darkest.

Reply:
> > but the scale would be consistent between colours so we'd know that towards 1000 is darker and towards 0 is lighter.
> 
> If that were the case I'd think we'd use adjacent numbers. Or is the idea to build in gaps in case we want in-between colors in the future?

Exactly